nucliadb_utils/src/nucliadb_utils/storages/gcs.py

```diff
@@ -484,15 +470,15 @@ def __init__(
         self.source = CloudFile.GCS
         self.deadletter_bucket = deadletter_bucket
         self.indexing_bucket = indexing_bucket
-        self.bucket = bucket
+        self.bucket = bucket or ""
```

Comment:
Is this safe? In which case we need a GCSStorage instance without knowing the bucket?